neqo-transport/src/connection/mod.rs

```diff
@@ -373,6 +373,8 @@ impl Connection {
     /// A long default for timer resolution, so that we don't tax the
     /// system too hard when we don't need to.
     const LOOSE_TIMER_RESOLUTION: Duration = Duration::from_millis(50);
+    /// The SCONE indicator.
+    const SCONE_INDICATION: &[u8] = &[0xc8, 0x13];
```

Comment:
I can't find these constants, nor the client -> server indicator mechanism, in the SCONE editors copy, SCONE draft 2, nor the TRAIN draft. Can you point me in the right direction?

Reply:
They are in pull requests for the moment: 

https://github.com/ietf-wg-scone/scone/pull/51 for version
https://github.com/ietf-wg-scone/scone/pull/49 for indication
https://github.com/ietf-wg-scone/scone/pull/50 for transport parameter (now merged)